frontend/test/metabase/scenarios/collections/permissions.cy.spec.js

```diff
@@ -256,6 +261,23 @@ describe("collection permissions", () => {
                 // We reverted the dashboard to the state prior to adding any cards to it
                 cy.findByText("This dashboard is looking empty.");
               });
+
+              it("should be able to revert the question", () => {
+                // It's possible that the mechanics of who should be able to revert the question will change, but for now that's not possible for user without data access
```

Comment:
```suggestion
                // It's possible that the mechanics of who should be able to revert the question will change, but for now that's not possible for user without data access - likely when #11719 is fixed
```

Reply:
I meant more about figuring out will users without data permissions be able to revert the question/dashboard. Are we going to be able to determine will the revert change the query or not, etc.

I talked with @camsaul about this offline and he pointed me to the conversation in https://github.com/metabase/metabase/issues/15131 (which might influence some of the outcomes we're talking about here).